style.css

```diff
@@ -0,0 +1,91 @@
+@media (min-width: 320px) and (max-width: 768px) {
+  .header-top {
+    display: flex;
+  }
+
+  .header-img-left {
+    width: 111px;
+    height: 74px;
+    margin-left: 0;
+  }
+
+  .header-nav-lg {
+    width: 20px;
+    height: 14px;
+    color: #c4c4c4;
+    margin-left: 174px;
+  }
+
+  .header-nav-lg a {
+    color: #c4c4c4;
+  }
+
+  .about-container {
+    display: flex;
+    flex-direction: column;
+    width: 288px;
```

Comment:
- [ ] _[OPTIONAL]_ Setting a fixed width is creating an unnecessary space in your site please fix that.